src/v2i-hub/TimPlugin/src/DsrcBuilder.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * DsrcBuilder.cpp
+ *
+ *  Created on: Sep 25, 2014
+ *      Author: ivp
+ */
+
+#include <string>
+#include <sstream>
+#include <OCTET_STRING.h>
+#include <ITIScodesAndText.h>
+
+#include "DsrcBuilder.h"
+#include "TimeHelper.h"
+
+void DsrcBuilder::AddCurveSpeedAdvisory(TiDataFrame *frame, unsigned int speedLimit)
```

Comment:
This needs to be cleaned up because the name seems to indicate a curve speed warning, delete if this was just a copy and paste.